add symfony 7 to composer

Scarbous opened the pull request:
update artifact action to v4
force symfony translation version in ci

Scarbous committed:
add symfony 7 to composer
update artifact action to v4
force symfony translation version in ci

Scarbous committed:
add prefer to ci
add php 8.4 to compsoer and cleanUp

Scarbous committed:
fix with symfony/translation version

Scarbous committed:
fix prefer
cache composer cachDir
switch to simple-phpunit

Scarbous committed:
switch from symfony 7.0 to 7.1

Scarbous committed:
add missing matrix.prefer

Scarbous committed:
fix Multiple uploads to the same named Artifact

Scarbous committed:
add min versions for symfony/phpunit-bridge

Scarbous committed:
fix version string

Scarbous committed:
try composer exec simple-phpunit

steveWinter commented:
Hi @Scarbous thanks for your work on this bundle. Any chance you might be able to merge this PR so that we can use this bundle with SF7?

google-labs-jules[bot] committed:
feat: Add support for Symfony 7

This commit updates the package to be compatible with Symfony 7.

The following changes were made:
- The `composer.json` file was updated to allow Symfony 7.0.
- The CI configuration was updated to test against Symfony 5.4, 6.4, and 7.0.
- All tests were run and passed for the supported Symfony versions.

Scarbous committed:
Merge pull request #10 from netlogix/feature/symfony-7-compatibility

feat: Add support for Symfony 7

Scarbous committed:
feat: update github actions to latest version

google-labs-jules[bot] committed:
fix: Resolve artifact conflict in CI

This commit resolves an artifact conflict in the CI workflow. The issue was caused by multiple jobs trying to upload an artifact with the same name.

The following changes were made:
- The `upload-artifact` steps in the `test` job were updated to use unique names for each artifact.
- The `download-artifact` step in the `coverage_report` job was updated to use a pattern to download all coverage artifacts.
- All `actions/upload-artifact` and `actions/download-artifact` were updated to `v4`.

Scarbous committed:
Merge branch 'update-test-pipeline' into feature/symfony-7-compatibility

Scarbous committed:
Merge pull request #11 from netlogix/feature/symfony-7-compatibility

fix: Resolve artifact conflict in CI

Scarbous committed:
fix: adjust download-artifact config
